src/main/java/org/prebid/server/analytics/AnalyticsReporterDelegator.java

```diff
@@ -72,4 +90,79 @@ public AnalyticsReporterDelegator(List<AnalyticsReporter> delegates,
                     tcfContext.getConsentString(), delegates);
         }
     }
+
+    private <T> void validateEvent(T event) {
+        if (event instanceof AuctionEvent) {
+            validateExtPrebidAnalytics((AuctionEvent) event);
+        }
+    }
+
+    private void validateExtPrebidAnalytics(AuctionEvent auctionEvent) {
```

Comment:
i'd say it is not a validation but rather just logging of unknown adapters.
Probably would be nice to rename like `logUnknownAdapters(...)` or so.